File: Command/PHP/Image.php
<?php

class Image {
    private int $imagePosition;
    private float $imageSize;

    public function __construct(int $imagePosition, float $imageSize) {
        $this->imagePosition = $imagePosition;
        $this->imageSize = $imageSize;
    }

    public function rotate(int $degree) {
        $this->imagePosition = abs(($this->imagePosition + $degree)) % 360;
    }

    public function resize(int $percentage) {
        $this->imageSize *= ($percentage / 100);
    }

    public function inspect() {
        echo "
            Image Position  : {$this->imagePosition}
            Image Size      : {$this->imageSize}
        ";
    }
}

File: Command/Python 3/Image.py
class Image:
  def __init__(self, imagePosition, imageSize):
    self.__imagePosition = imagePosition
    self.__imageSize = imageSize
  
  def rotate(self, degree):
    self.__imagePosition = abs(self.__imagePosition + degree) % 360
  
  def resize(self, percentage):
    self.__imageSize *= (percentage / 100)
  
  def inspect(self):
    print("""
    Image position: {0}
    Image size    : {1}
    """.format(
      self.__imagePosition,
      self.__imageSize
    ))